styles/colors-silicon-night.less

```diff
@@ -103,9 +102,10 @@
 // Item
 // ---------------------------------------
 @agate-item-color:           @agate-text-color;
-@agate-item-label-color:     @agate-accent;
-@agate-item-focus-border-color: @agate-accent;
+@agate-item-border-color: rgba(255, 255, 255, 0.4);
+@agate-item-focus-border-color: transparent;
 @agate-item-focus-color:     @agate-spotlight-color;
+@agate-item-active-bg-color:  rgba(255, 255, 255, 0.1);
```

Comment:
When focusing an item, the text color changes from black to white. Did you intend to do it?